### PR TITLE
code health rc2test

### PR DESCRIPTION
**Pull request by pauli-oracle**
- [x] tests are added or updated

Update rc2test to use the test infrastructure

### CODE REVIEW

File: test/rc2test.c [hunk 1]
```diff
@@ -52,48 +39,30 @@ static unsigned char RC2cipher[4][8] = {
     {0x50, 0xDC, 0x01, 0x62, 0xBD, 0x75, 0x7F, 0x31},
 };
 
-int main(int argc, char *argv[])
+static int test_rc2(const int n)
 {
-    int i, n, err = 0;
+    int testresult = 1;
```
Comment (kroeckx) [inline test/rc2test.c]: Do we have a define to say OK / NOK?

Comment (pauli-oracle) [inline test/rc2test.c]: I don't think there is one.

Comment (richsalz) [inline test/rc2test.c]: We don't.

**Comment by richsalz** [comment]
commit 05d7752; thanks.

**Comment by pauli-oracle** [comment]
Code health tag?

**Comment by richsalz** [comment]
are you asking if we need the label?  since this was merged, it's okay.  the label is just to get dev-team attention.

**Comment by pauli-oracle** [comment]
I'll need the tags to do the summary later in the week.
Otherwise I won't be able to find them easily.  I.e. I'll miss some :)

**Comment by levitte** [comment]
Added

**Comment by pauli-oracle** [comment]
Thanks.

**Pull request closed without merging**